NineChronicles.Standalone/NineChroniclesNodeService.cs

```diff
@@ -159,6 +159,7 @@ public class NineChroniclesNodeService
                         {
                             Log.Debug("Start mining.");
                             await miner.MineBlockAsync(properties.MaximumTransactions, cancellationToken);
+                            await Task.Delay(blockInterval, cancellationToken);
```

Comment:
I wonder it doesn't cost any time for mining in dev mode.

Reply:
Its difficulty is at most 2, so it takes almost no time.